testing/asserts.ts

```diff
@@ -641,66 +641,86 @@ export function assertIsError<E extends Error = Error>(
  * callback which will be passed the error, usually to apply some custom
  * assertions on it.
```

Comment:
Was thinking to remove this bit about error callback from the documentation not to encourage developers to use it, because we're deprecating this parameter. What do you think?